baggers.js

```diff
@@ -13352,39 +13352,67 @@ var data = {
         "mail": "fpot@lucca.fr"
       }
     },
-    {
```

Comment:
Harg sorry my idea is getting me crazy and doesn't show me the format change let me fix that :(